modules/ROOT/pages/6.7-release-notes.adoc

```diff
@@ -296,6 +296,11 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 === Numeric input in toolbar items did not disable when a switching from edit to read-only mode
 // TINY-10129
+In previous versions of {productname}, an issue related to the `fontsizeinput` was identified, when the Editor's configuration was set to **readonly** mode.
+
+This issue was due to the `Changenode` event not being fired when switching modes, which prevented the event listener responsible for disabling the `fontsizeinput` button from triggering. Consequently this caused the toolbar button to appear to be interactable even while the editor was set to **readonly** mode.
+
+{productname} 6.7, addresses this fix by listening to the "ChangeMode" event in addition to the previous event, ensuring that the missing event is now properly handled. As a result, the bug no longer exists, and the `fontsizeinput` button is correctly disabled when the Editor's configuration is set to **readonly** mode.
```

Comment:
```suggestion
Previously, when a {productname} editor instance was set to *read-only* mode, because a `Changenode` event was not fired when switching modes, the text-entry field in the `fontsizeinput` toolbar object did not present as disabled and the field’s value could be changed.

{productname} 6.7, addresses this by listening to the `ChangeMode` event in addition to the previous event. This ensures the missing event is now properly handled.

As a consequence, the `fontsizeinput` text-entry field presents as disabled when a {productname} editor instance is set to *read-only* mode, as expected.

NOTE: this was a display error only. Previously, when the editor was set to *read-only* mode, text could be entered into the `fontsizeinput` text-entry field. Setting this field to a new value in this circumstance did not, however, have any effect on selected material in a {productname} document, nor on text added at an insertion point after the {productname} instance was switched away from *read-only* mode.
```
Suggested edit.